app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/VatGroupController.scala

```diff
@@ -37,11 +40,14 @@ class VatGroupController @Inject() (mcc: MessagesControllerComponents, vatGroupV
       .fold(
         formWithErrors => BadRequest(vatGroupView(formWithErrors, service)),
         yesNoAnswer =>
-          if (yesNoAnswer.isNo)
-            // TODO - need service url param here
-            Redirect(EmailController.form(service))
-          else
-            Redirect(routes.VatGroupsCannotRegisterUsingThisServiceController.form(service))
+          (yesNoAnswer.isYes, featureFlags.edgeCaseJourney) match {
+            case (false, true)  => Redirect(VatDetailsController.createForm(service))
+            case (false, false) => Redirect(EmailController.form(service))
+            case (true, _) =>
+              Redirect(
+                routes.VatGroupsCannotRegisterUsingThisServiceController.form(service)
```

Comment:
We don't need the routes. on this. (Also that's such a long name for a class, is there a meaningful import alias?)